src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -269,13 +312,29 @@ def create_replica(cmd, client, store_name, name, location, resource_group_name=
                                replica_creation_parameters=replica_creation_params)
 
 
-def delete_replica(cmd, client, store_name, name, resource_group_name=None):
+def delete_replica(cmd, client, store_name, name, yes=False, resource_group_name=None):
```

Comment:
This already had a confirmation, right?

Reply:
Yes the delete command already had a confirmation.